mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/ui/play/PlayScreenTest.kt

```diff
@@ -14,13 +16,16 @@ open class TestPlayScreenState(
     val onGameItemClickAction: (ChessMatch) -> Unit,
     override val matches: List<ChessMatch>
 ) : PlayScreenState<ChessMatch> {
+  override val user: AuthenticatedUser = mockk()
+  override fun onLocalGameClick() {}
+  override fun onOnlineGameClick() {}
   override fun onMatchClick(match: ChessMatch) {
     onGameItemClickAction(match)
   }
 
-  override fun onNewGameClick() {
-    onNewGameClickAction()
-  }
+  // override fun onNewGameClick() {
+  //  onNewGameClickAction()
+  // }
```

Comment:
Should this be removed ?

Reply:
Fixed in [368c6ca](https://github.com/epfl-SDP/android/pull/226/commits/368c6cae7026f7ef2f5e3074f763f180786a9cb1)